packages/plugin-ext/src/main/browser/plugin-icon-theme-service.ts

```diff
@@ -387,7 +409,8 @@ export class PluginIconTheme extends PluginIconThemeDefinition implements IconTh
     protected collectSelectors(
         associations: RecursivePartial<PluginIconsAssociation>,
         accept: (definitionId: string, ...icons: string[]) => void
-    ): void {
+    ): { hasSpecificFileIcons: boolean; coveredLanguages: { [languageId: string]: boolean } } {
+        const iconInfo = { hasSpecificFileIcons: false, coveredLanguages: {} as { [languageId: string]: boolean } };
```

Comment:
Suggestion: We should probably have a more explicit type for this
```suggestion
    ): PluginLanguageIconInfo {
        const iconInfo: PluginLanguageIconInfo  = {
            hasSpecificFileIcons: false, 
            coveredLanguages: {}
        };
```

Reply:
Good point, I fixed it.